addons/l10n_pl/data/account_tax_report_data.xml

```diff
@@ -172,22 +172,22 @@
         <field name="name">Podatek - Do przeniesienia</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="3"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
     </record>
 
     <record id="account_tax_report_line_nad_naleznym" model="account.tax.report.line">
         <field name="name">Podatek - Nadwyżka naliczonego nad należnym</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="1"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
         <field name="parent_id" ref="account_tax_report_line_do_przeniesienia"/>
     </record>
 
     <record id="account_tax_report_line_do_US" model="account.tax.report.line">
         <field name="name">Podatek - Do wpłaty do US</field>
         <field name="report_id" ref="tax_report"/>
         <field name="sequence" eval="1"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
```

Comment:
Please add a code on `account_tax_report_line_podatek_razem_c` and use it here
See the description of the commit for more info https://github.com/odoo/enterprise/commit/5d7ed69e423d6e6814a5a185279afd3b634c531b